packages/jest/plugins/resolver.ts

```diff
@@ -1,3 +1,8 @@
+// allow tspaths to work within jest global files like setup/teardown
+process.env.TS_NODE_PROJECT =
+  process.env.TS_NODE_PROJECT || 'tsconfig.base.json';
+require('tsconfig-paths/register');
```

Comment:
We have a util function for this now: https://github.com/nrwl/nx/blob/master/packages/nx/src/utils/register.ts